src/domain/_common/components/Button.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+interface ButtonProps {
+  isDisabled: boolean;
+  type: 'normal' | 'text' | 'primary' | 'large';
+  onClick: () => void;
+  children?: React.ReactNode;
+}
+
+const buttonStyle = {
+  text: 'text-grey-buttontext disabled:text-grey-whitegray bg-[rgba(236, 236, 236, 0.4)] active:bg-grey-background',
```

Comment:
```suggestion
  text: 'text-grey-buttontext disabled:text-grey-whitegray bg-whitegray bg-opacity-40 active:bg-grey-background',
```
`tailwind.config.js` 에 정의된 스타일을 쓰는게 좋아보여요

Reply:
아.. 저게 마땅히 맞는 스타일이 없어서 그랬기는 했는데 일단은 고칠게요!

Reply:
text의 경우에는 특성상 텍스트만 보여지는 형태의 버튼이라 active를 제외한 나머지 bg 속성은 제외할게요!